### PR TITLE
FIX add more instructions to select models in parallel coordinate plot

### LINKED ISSUE #290: Quiz 9 question 3/4
Give more precise instructions on how to use the mouse to select the correct range to be able to answer the question.

### PR DESCRIPTION
**Pull request by glemaitre**
closes #290
see https://gitlab.inria.fr/learninglab/mooc-scikit-learn/mooc-scikit-learn-coordination/-/merge_requests/14

### CODE REVIEW

File: jupyter-book/tuning/parameter_tuning_automated_quiz.md [hunk 1]
```diff
@@ -111,14 +117,19 @@ lower than 0.8. What do have all these models in common?
...
 
 ```{admonition} Question
-Using the above plot, identify ranges of values for hyperparameter that
-always prevent the model to reach a test score higher than 0.86, irrespective
-of the other values:
+In the parallel coordinate plot shown above, select the models with a score
+higher than 0.85. You can select the range [0.85, max] by clicking and holding
+on the `mean_test_score` axis of the parallel coordinate plot.
+
+Identify ranges of values for hyperparameter that always prevent the model to
```
Comment (ogrisel) [inline jupyter-book/tuning/parameter_tuning_automated_quiz.md:129]:
```suggestion
Identify ranges of values for hyperparameters that always prevent the model to
```

File: jupyter-book/tuning/parameter_tuning_automated_quiz.md [hunk 2]
```diff
@@ -111,14 +117,19 @@ lower than 0.8. What do have all these models in common?
...
+reach a test score higher than 0.85, irrespective of the other values. In other
+words, which hyperparameter values are never used to get a good model
+(i.e. with `mean_test_score higher than 0.85).
```
Comment (glemaitre) [inline jupyter-book/tuning/parameter_tuning_automated_quiz.md:132]:
```suggestion
words, which hyperparameters values are never used to get a good model
(i.e. with `mean_test_score` higher than 0.85).
```
